src/metadata_generator.py

```diff
@@ -687,7 +772,11 @@ def handle_biosample(self, row: pd.Series) -> tuple:
         return emsl_metadata, biosample_id
 
     def check_for_biosamples(
```

Comment:
Can we update the name and documnetation of this method to be more descriptive of what it's doing?  Especially the output I don't quite understand.  I'm trying to understand why it needs a client id and secret.  To generate biosamples?  If that's true, let's make sure that the method name and documentation gives us a clue as to why it needs to mint ids.

Reply:
Much, thank you!  This looks ready to merge to me.